package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "mattermost-desktop",
+  "desktopName": "Mattermost.desktop",
```

Comment:
Assuming that, `desktopName` is needed only for `src/package.json`. It's actually bundled in the distributed packages. `/package.json` is used for development in almost cases.